README.md

```diff
@@ -1,8 +1,14 @@
 # Adept Augmentations
 
-Welcome to Adept Augmentation, the ultimate solution for those struggling with few shot Named Entity Recognition (NER) tasks!
+Welcome to Adept Augmentations, can be used for creating additional data in Few Shot Named Entity Recognition (NER) setting!
```

Comment:
```suggestion
Welcome to Adept Augmentations, which can be used for creating additional data in Few Shot Named Entity Recognition (NER) settings!
```
The sentence didn't really make much sense before. With the suggestion it's only slightly better.